api/src/test/java/com/github/eyefloaters/console/api/TopicsResourceIT.java

```diff
@@ -466,6 +471,104 @@ void testListTopicsSortedByUnknownField(String sortKey) {
             .body("data.id", contains(sortedIds));
     }
 
+    @Test
+    void testListTopicsPaginationLinksWithDefaultPageSize() throws MalformedURLException {
```

Comment:
This test would benefit from comments to explain which page you're requesting each time

Reply:
Comments added to each of the page requests about the requested page and the source of the link.